addons/mail/static/src/core/common/message.xml

```diff
@@ -143,30 +143,32 @@
             'mt-1': !env.inChatWindow and !ui.isSmall and !message.is_note,
             'my-n2': message.is_note,
             'invisible': !isActive,
-            'o-expanded': optionsDropdown.isOpen
+            'o-expanded': optionsDropdown.isOpen,
         }"
     >
         <div class="d-flex rounded-1 overflow-hidden" t-att-class="{
                 'flex-row-reverse': env.inChatWindow and isAlignedRight,
+                'flex-row': !(env.inChatWindow and isAlignedRight),
                 'bg-view shadow-sm': env.inChatWindow,
+                'list-group list-group-flush': ui.isSmall,
             }"
         >
             <t t-foreach="messageActions.actions.slice(0, quickActionCount)" t-as="action" t-key="action.id">
                 <t t-if="action.callComponent" t-component="action.callComponent" t-props="action.props"/>
-                <button t-else="" class="btn px-1 py-0 rounded-0" t-att-title="action.title" t-att-name="action.id" t-on-click.stop="action.onClick">
-                    <i class="fa fa-lg" t-att-class="action.icon"/>
+                <button t-else="" class="btn py-0 rounded-0" t-att-class="{ 'list-group-item border fs-4 px-1': ui.isSmall, 'px-1': !ui.isSmall }" t-att-title="action.title" t-att-name="action.id" t-on-click.stop="action.onClick">
+                    <i class="fa fa-lg" t-att-class="action.icon" t-attf-class="{{ ui.isSmall ? 'fa-fw': '' }}"/>
```

Comment:
I always prefer `t-att-class` with an object, it feels cleaner. We should stick to it every time it's possible.

Reply:
We should just use whatever works and is the most straight-forward for the current use.

Ternary is totally fine when it's just if/else. If you need more conditionals, then object syntax makes more sense.
I wouldn't just blindly make object syntax of an equivalent ternary, because this is more verbose.